docs/role_variables.md

```diff
@@ -96,6 +96,7 @@ _Note: The above options are intended to provide users with flexibility. This al
 |--------------------|---------------|------------------------------|
 | `sensu_user_name`    | root        | The name of the Sensu service user |
 | `sensu_group_name`   | root        | The name of the Sensu service user's primary group |
+| `sensu_package`      | sensu       | The name of the Sensu package. Can optionally include a version (sensu=0.20.3-1) |
```

Comment:
This is accurate for Debian/Amazon Linux/Ubuntu but for Redhat/CentOS/FreeBSD, the role currently uses the `sensu_pkg_version` variable to help expose which version should get installed. 

We should probably fix that up so that `sensu_pkg_version` works for all OS's and then ensure that it's documented here. 

But for the time being, thanks! I'll merge this in